public/views/FileTransfer.svelte

```diff
@@ -453,13 +478,26 @@
   </main>
 
   <Fab
+    name="fab"
+    id="addButton"
     icon="icon-add"
     disabled={!isSelectorEnabled}
     text="Add File"
     on:click={() => document.getElementById('inpFiles').click()}
   >
   </Fab>
 
+  <button
+    id="drop_zone"
+    class="dropzone"
+    disabled={!isSelectorEnabled}
+    on:drop={dropHandler}
+    on:dragover={dragOverHandler}
+    on:dragleave={dragLeaveHandler}
+    on:click={() => document.getElementById('inpFiles').click()}
+    >Drop files or click
+  </button>
```

Comment:
The dropzone has removed the original file selector button on desktops.
Try making the full screen as the dropzone for files, while still keeping the old file selector button even on desktops.